api/src/main/java/org/openmrs/BaseConceptMap.java

```diff
@@ -79,31 +79,45 @@ public void setChangedBy(User changedBy) {
 	 */
 	@Override
 	public Date getDateCreated() {
-		return dateCreated;
+		if(dateCreated == null) {
+			return null;
+		}
+		return (Date) dateCreated.clone();
 	}
 	
 	/**
 	 * @param dateCreated the dateCreated to set
 	 */
 	@Override
 	public void setDateCreated(Date dateCreated) {
-		this.dateCreated = dateCreated;
+		if(dateCreated == null) {
+			this.dateCreated = null;
+			return;
+		}
+		this.dateCreated = (Date) dateCreated.clone();
```

Comment:
Hii @PermissionError, have a look at [this](http://www.informit.com/articles/article.aspx?p=31551&seqNum=2) article, using `clone()` method in setters may possess some security risks :)

Reply:
Thanks for the tip, very valuable indeed. PR amended :)